.gitignore

```diff
@@ -11,4 +11,8 @@ pkg/
 .kubernetes-manifests-*/
 .project
 .eclipse.buildship.core.prefs
-release/wi-kubernetes-manifests.yaml
\ No newline at end of file
+.terraform
+.terraform.lock.hcl
+release/wi-kubernetes-manifests.yaml
+terraform.tfstate
```

Comment:
# .tfstate files
*.tfstate
*.tfstate.*
instead of the terraform.tfstate and terraform.tfstate.backup